docs/weave/design/03-permissions.md

```diff
@@ -0,0 +1,156 @@
+---
```

Comment:
'In this section (03-permissions) the writer toggles between I and we. This should be standardised to We.'

Reply:
Nice catch